components/collapsible/collapsible.test.js

```diff
@@ -115,18 +145,24 @@ describe('Dialtone vue Collapsible Component Tests', function () {
         await wrapper.setProps({ open: true });
       });
 
-      it('content is expanded', function () {
-        assert.isTrue(contentElement.isVisible());
+      it('content is expanded when open = true', function () {
```

Comment:
Don't need to describe state here since you've already done it in the parent describe. Just "content is expanded".

Reply:
Done